pynicotine/gtkgui/ui/chatrooms.ui

```diff
@@ -94,6 +94,21 @@
                         </style>
                       </object>
                     </child>
+                    <child>
+                      <object class="GtkToggleButton" id="user_list_button">
+                        <property name="tooltip-text" translatable="yes">Show Room Users</property>
+                        <property name="visible">True</property>
+                        <child>
+                          <object class="GtkImage">
+                            <property name="icon-name">sidebar-show-symbolic</property>
+                            <property name="visible">True</property>
+                          </object>
+                        </child>
+                        <style>
+                          <class name="image-button"/>
+                        </style>
+                      </object>
+                    </child>
```

Comment:
Consider making the log toggle a button with `media-record-symbolic` or `media-playlist-play`, and moving the Sidebar button right to the end so that it is directly adjacent to the controlled widget.

Reply:
I'll have to try this with a few different icons in icon themes to see what's the most understandable. It won't happen in this PR though.